tests/testapp/test_helpers.py

```diff
@@ -15,35 +15,37 @@ def __new__(cls, id_):
         return super(WithId, cls).__new__(cls, id_, 'type name')
 
 
-def run_testcase_with_django_runner(
```

Comment:
@zsoldosp I'm not too happy with the code myself, but I think my brain needed the logical separation of initializing the thing and running it (or basically breaking it into smaller pieces). Accessing the test_runner instance from a dictionary that is returned from run_testcase_with_django_runner confused me, since it's not a result of "running".